### ISSUE TITLE
Docs request: Reframe the "running an analysis module" doc

### ISSUE THREAD

**Issue by sjspielman** [AlexsLemonade/OpenScPCA-analysis#475]
### What is the documentation improvement or update you wish to see?

While looking over https://openscpca.readthedocs.io/en/latest/contributing-to-analyses/analysis-modules/running-a-module, I wondered if we achieved the right scope for this page. To me, the title "running an analysis module" implies does on how someone else could run a module. Yet, we mostly discuss writing a script that enables running the analysis module, which is more a concept related to authors establishing a module, vs contributors running a module. In addition, we don't include any discussion of setting up an environment to run a module besides simply stating "the readme should have all the info."

### Is there any additional context you would like to provide?

Just tagging in @dvenprasad for any thoughts on whether this page suits us and discuss what updates/reorgs we might benefit from.

### If this request is for an existing documentation page, please provide the link here.

https://openscpca.readthedocs.io/en/latest/contributing-to-analyses/analysis-modules/running-a-module/

**Comment by dvenprasad** [comment]
Yes, I believe we made the decision to redirect folks to the module instead of explaining in the docs because the running it might differ based on the module. I do not think we should change that. 

But I do think we should tell people to include directions to install packages like we do in the example module for new packages. (https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/analyses/hello-R/README.md#software-requirements)

**Comment by sjspielman** [comment]
>  I do not think we should change that.

I agree, we should not change that! I did not clearly express what I wanted to emphasize - that most of the page is about "creating a script to run a module," and not about running a module itself. 

I'm therefore proposing 2 things:
- Add a lil information about installing module-specific environments 
- _Move_ the content about writing a script out of this document, and into (I think??? But not sure!) "creating an analysis module"
